sale_order_type_automation/README.rst

```diff
@@ -11,7 +19,6 @@ Sale Order Type Automation
     * Create / Validate Payment
```

Comment:
add a blank line betwent the ordered and unordered list to avoid warning raised in odoo log only when trying to view the module description